packages/cli/generators/repository/templates/src/repositories/repository-crud-default-template.ts.ejs

```diff
@@ -0,0 +1,12 @@
+import {<%= repositoryTypeClass %>}, juggler} from '@loopback/repository';
+import {<%= modelName %>} from '../models';
+import {inject} from '@loopback/core';
+
+export class <%= className %>Repository extends <%= repositoryTypeClass %><
+  <%= modelName %>, <%= idType %>> {
```

Comment:
I think this is not going to pass `prettier` linter and should be changed to the following:

```ejs
export class <%= className %>Repository extends <%= repositoryTypeClass %><
  <%= modelName %>, 
  <%= idType %>,
> {
```